README.md

```diff
@@ -16,6 +16,16 @@ npx @oxlint/migrate <optional-eslint-flat-config-path>
 
 When no config file provided, the script searches for the default eslint config filenames in the current directory.
 
+### Options
+
+| Options     | Description                                      |
+| ----------- | ------------------------------------------------ |
+| `--upgrade` | \* Upgrade existing .oxlintrc.json configuration |
+
+\* WARNING: When some `categories` are enabled, this tools will enable more rules with the combination of `plugins`.
```

Comment:
@Boshen I think this should be not much of a problem. 

After working on it over days I think the Option should be renamed to `--merge`. `--upgrade` can be a bit of confusing. Maybe the user thinks it upgrades the tool and not the configuration.